<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225);</string>
   </property>
   <widget class="QLabel" name="m_oLabelWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>340</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Popular Tools</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnAddPrefixSuffix">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Prefix/Suffix into Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnAddRemoveLineBreaks">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add/Remove Line Breaks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnDelimitedExtractor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Delimited Column Extractor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnFindAndReplace">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Find and Replace Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnJoinMerge">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Join/Merge Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnRemoveDuplicateLines">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Remove Duplicate Lines</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnRemoveEmptyLines">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Remove Empty Lines</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnRemoveExtraSpaces">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Remove Extra Spaces</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnRemoveLinesContaining">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Remove Lines Containing..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_oBtnSortTextLines">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sort Text Lines</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
